docs/howto/grape_integration.md

```diff
@@ -0,0 +1,40 @@
+The AMS grape formatter relies on the existence of `env['grape.request']` which is implemeted by `Grape::Middleware::Globals`. You can meet his dependency by calling it before mounting the endpoints.
```

Comment:
Missing back to guides link at top, see other pages for example

Also please link from integrations table

I know this is mentioned elsewhere